code/zoltar_scripts/compare_repo_to_zoltar.py

```diff
@@ -0,0 +1,44 @@
+from zoltpy import util
+import os
+
+# 1 Meta info 
+
+## Set connections to Zoltar project and github repo
+project_name = 'ECDC European COVID-19 Forecast Hub'
+project_obj = None
+project_timezeros = []
+conn = util.authenticate()
+url = 'https://github.com/epiforecasts/covid19-forecast-hub-europe/tree/master/data-processed/'
```

Comment:
It might be good to keep a list somewhere of things that would be good to store in a [config file](https://github.com/epiforecasts/covid19-forecast-hub-europe/pull/511) but aren't a priority to implement.

Reply:
@kathsherratt, you can now use the `repo_url` for the config file `forecasthub.yml` if you wish.